app/serializers/about_serializer.rb

```diff
@@ -35,4 +37,24 @@ def include_stats?
   def stats
     object.class.fetch_cached_stats || Jobs::AboutStats.new.execute({})
   end
+
+  def can_see_site_contact_details
+    scope.can_see_site_contact_details?
```

Comment:
I don't think the logic should be pushed into the guardian.. yet. Contact details is very specific to this serializer so moving the logic into guardian makes it more confusing IMO.